cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -17,7 +17,7 @@ <h6 class="float-right">
 <div class="card shadow-sm mb-3">
   <div class="card-header">
     <span class="oi oi-bar-chart mr-2 ml-n1"></span>
-    Publish requests
+    Publish requests (total number of TEKs published)
```

Comment:
this chat I think we need to separate - because the units are the same.

How about

`TEK Publish Metrics`